src/Project.js

```diff
@@ -633,6 +637,19 @@ class Project extends DirItem {
         } else {
             exitValue = this.resultStats.errors > 0 ? 2 : ((this.resultStats.warnings > 0) ? 1 : 0);
         }
+
+        if (this.options.opt.output) {
+            if (typeof this.formatter.writeSummaryToFile === 'function') {
+                let summary = this.formatter.writeSummaryToFile(this.project.name, totalTime, this.fileStats, this.resultStats, score) ;
+                resultSet = summary+resultSet;
+            }
+            if (typeof this.formatter.completeFile === 'function') {
+                resultFull = this.formatter.completeFile(resultSet);
```

Comment:
Same thing here. `formatHeader()` and `formatFooter()` should defined in the Formatter superclass in `ilib-lint-common`. The current output of the linter (stats and score, etc.) should be the default footer if the current formatter does not define one.

Reply:
Maybe create a formatSummary() and formatFooter() function in the built-in AnsiConsoleFormatter as the place where the current summary, etc. is created?

Reply:
As we discussed at the last meeting, I created a new method `outputFormat()` in the Formatter class. I also updated it in ilib-lint-common. (https://github.com/iLib-js/ilib-lint-common/pull/16) and all kinds of formatHeader() , formatFotter() and formatSummary() are handled inside formatter's outputFormat()